samples/dnn/yolo_object_cam.cpp

```diff
@@ -0,0 +1,184 @@
+	{
+		cerr << "Can't load network by using the following files: " << endl;
+		cerr << "cfg-file:     " << modelConfiguration << endl;
+		cerr << "weights-file: " << modelBinary << endl;
+		cerr << "Models can be downloaded here:" << endl;
+		cerr << "https://pjreddie.com/darknet/yolo/" << endl;
+		exit(-1);
+	}
+
+	Mat frame;
+	VideoCapture cap(cam);
+	if (!cap.isOpened()) return -1;
+
+	int frame_width=  416;
+	int frame_height=  416;
+
+	
+	bool grabFrame = true;
+	while (grabFrame) 
+	{
+		cap >> frame;
+
+		//cv::Mat frame = cv::imread(parser.get<string>("image"));
+
+		//! [Resizing without keeping aspect ratio]
+		cv::Mat resized;
+		cv::resize(frame, resized, cv::Size(network_width, network_height));
+		//! [Resizing without keeping aspect ratio]
+
+		//! [Prepare blob]
+		Mat inputBlob = blobFromImage(resized, 1 / 255.F); //Convert Mat to batch of images
+		//! [Prepare blob]
+
+		//! [Set input blob]
+		net.setInput(inputBlob, "data");                //set the network input
+		//! [Set input blob]
+
+		//! [Make forward pass]
+		cv::Mat detectionMat = net.forward("detection_out");	//compute output
+		//! [Make forward pass]
+
+		float confidenceThreshold = parser.get<float>("min_confidence");
+		for (int i = 0; i < detectionMat.rows; i++)
+		{
+			const int probability_index = 5;
+			const int probability_size = detectionMat.cols - probability_index;
+			float *prob_array_ptr = &detectionMat.at<float>(i, probability_index);
+
+			size_t objectClass = std::max_element(prob_array_ptr, prob_array_ptr + probability_size) - prob_array_ptr;
+			float confidence = detectionMat.at<float>(i, (int)objectClass + probability_index);
+
+			if (confidence > confidenceThreshold)
+			{
+					string sLabel = returnLabel(objectClass,mylist );
+					float x = detectionMat.at<float>(i, 0);
+					float y = detectionMat.at<float>(i, 1);
+					float width = detectionMat.at<float>(i, 2);
+					float height = detectionMat.at<float>(i, 3);
+					float xLeftBottom = (x - width / 2) * frame.cols;
+					float yLeftBottom = (y - height / 2) * frame.rows;
+					float xRightTop = (x + width / 2) * frame.cols;
+					float yRightTop = (y + height / 2) * frame.rows;
+
+					std::cout << "Class: " << objectClass << " " <<sLabel<<std::endl;
+					std::cout << "Confidence: " << confidence << std::endl;
+					std::cout << " " << xLeftBottom << " " << yLeftBottom << " " << xRightTop << " " << yRightTop << std::endl;
+
+					Rect object((int)xLeftBottom, (int)yLeftBottom, (int)(xRightTop - xLeftBottom), (int)(yRightTop - yLeftBottom));
+					assetRoi(object,frame);
+						
+					rectangle(frame, object, Scalar(0, 255, 0),2);
+					cv::Mat roi = frame(object);
+					cv::Mat color(roi.size(), CV_8UC3, cv::Scalar(0, 0, 0)); 
+					double alpha = 0.3;
+					cv::addWeighted(color, alpha, roi, 1.0 - alpha , 0.0, roi); 
+		 
+					putText(frame, sLabel, Point(object.x, object.y+12),FONT_HERSHEY_PLAIN, 1 ,Scalar::all(255)); 
+			
+			}
+		}
+		 
+		imshow("GPU", frame); //waitKey();
```

Comment:
"GPU" -> "Result" / "YOLO"
(by default GPU acceleration is not enabled)

Reply:
Done